docs/site/decorators/Decorators_openapi.md

```diff
@@ -191,6 +191,33 @@ You can find specific use cases in
 _The parameter location cookie is not supported yet, see_
 _(https://github.com/strongloop/loopback-next/issues/997)_
 
+### Parameter decorator for objects
+
+One of the parameter decorator types, `@param.query.object` is applied to
+generate an OpenAPI definition for query parameters with JSON values.
+
+The generated definition supports the `url-encoded` style specified in Open API.
```

Comment:
> supports the `url-encoded` style specified in OpenAPI. 

Could you explain more about what does "url-encoded style specified in OpenAPI mean"? IIUC that style describes the syntax of an URL, but has nothing to do with OpenAPI spec.

Reply:
agreed, Open API does not openly state an url-encoded style , but it defines a certain specification that allows url-encoded payload for query params. I will explain more about this.